Lists/scams.txt

```diff
@@ -4,6 +4,7 @@ strearmcommunity.ru
 steamcomunitty.ru
 streancommunuty.ru
 steamcommunlty.shop
+stearncornmunlty.ru
```

Comment:
This list is NOT ordered alphabetically. Please add new entries to the bottom of the file. It's chronological.